test-output/old/Default suite/methods-alphabetical.html

```diff
@@ -0,0 +1,6 @@
+<h2>Methods run, sorted chronologically</h2><h3>&gt;&gt; means before, &lt;&lt; means after</h3><p/><br/><em>Default suite</em><p/><small><i>(Hover the method name to see the test class name)</i></small><p/>
```

Comment:
Should the output of the tests be checked in? I'm not 100% sure that's what this is, but if so I think this test-output folder could also be ignored.